### ISSUE TITLE
Search Refunds by Creation Date

### ISSUE THREAD

**Issue by jmawebtech** [stripe/stripe-dotnet#1382]
Hi,

I am trying to search refunds by creation date. In my old plugin, here is the URL that I would build:

```c#
public List<Refund> SearchRefunds(DateTime beginTime, DateTime endTime)
{
    string url = String.Format("{0}/v1/refunds?created%5Bgte%5D={1}&created%5Blte%5D={2}&limit=100&expand[]=data.charge&expand[]=data.charge.customer&expand[]=data.charge.invoice", m_BaseUrl, ConvertToTimestamp(beginTime), ConvertToTimestamp(endTime));
}
```

How do I pull refunds using the date created field with your plugin?

**Comment by ob-stripe** [comment]
Hey @jmawebtech, something like this should work:

```c#
var refundService = new RefundService();
var listOptions = new RefundListOptions
{
  CreatedRange = new DateRangeOptions
  {
    GreatherThanOrEqual = beginTime,
    LessThanOrEqual = endTime,
  },
};
listOptions.AddExpand("data.charge");
listOptions.AddExpand("data.charge.customer");
listOptions.AddExpand("data.charge. invoice");

var refunds = refundService.List(listOptions);
```

...except that I just realized that the `RefundListOptions` class doesn't have the `CreatedRange` attribute. That's an oversight, we'll fix it shortly and release an update.

**Comment by ob-stripe** [comment]
This is fixed as of version 21.1.0.
